.github/workflows/test.yml

```diff
@@ -67,7 +67,7 @@ jobs:
           authToken: "${{ secrets.CACHIX_AUTH_TOKEN }}"
       - name: Run tests
         run: |
-          nix run .#marlowe-integration-tests --show-trace -- --strict
```

Comment:
The reason why I renamed this is that the flake output would shadow the testsuite named `marlowe-integration-tests` in the `marlowe-integration-tests/marlowe-integration-tests.cabal`